src/opentom_skel/bin/navit.sh

```diff
@@ -1,19 +1,23 @@
 #! /bin/sh
 
+if [ x`cat /proc/barcelona/gpstype` == x3 ]
+then 
+    ln -sf `cat /proc/barcelona/gpsdev` /dev/gpsdata
+else
 ls $DIST/bin/gltt || {
 	echo "You need TomTom(tm) gltt\nPlease read documentation." | flmessage -s -t "Can't start Navit"
 	exit
 }
+if [ -z `pidof gltt` ]; then rc.gltt start; fi
+echo EnableRawGPSOutput >/dev/gps
+fi
 
-export LANG=fr_FR.utf8
+export LANG=de_DE.utf8
```

Comment:
Yes navit NLS don`t work. I think it is a glibc problem. Installing gettext or use the integrated gettext fail also.  We should use US or comment it out.
I work on a script to translate the navigation announce at source level.